packages/taco/test/conditions/context.test.ts

```diff
@@ -145,10 +148,11 @@ describe('context', () => {
     });
 
     it('rejects on a missing signer', () => {
-      const conditionObj = {
+      const conditionObj: ContractConditionProps = {
         ...testContractConditionObj,
         returnValueTest: {
-          ...testReturnValueTest,
+          index: 0,
```

Comment:
See prior comments about `index` not being applicable.

Also applies to other conditions after this.